CMakeLists.txt

```diff
@@ -21,3 +21,8 @@ add_subdirectory(subsys)
 add_subdirectory(modules)
 add_subdirectory(drivers)
 add_subdirectory(tests)
+
+if(CONFIG_SPM AND NOT CONFIG_IS_SPM)
+	include(${CMAKE_BINARY_DIR}/spm/shared_vars.cmake)
+	zephyr_compile_definitions(-DCONFIG_SPM_FPU_CONFIG=${spm_CONFIG_FPU})
```

Comment:
This will print a build error at stage 6 as:
```
[6/253] cd /projects/github/ncs/nrf/build_pristine/modules/nrf && /usr/bin/python3.8 -c "print ( 'Mismatch between fpu configurations in application and spm, CONFIG_FPU=y, spm_CONFIG_FPU=n' ) ; exit ( 1 ) "
FAILED: modules/nrf/CMakeFiles/spm_fail 
cd /projects/github/ncs/nrf/build_pristine/modules/nrf && /usr/bin/python3.8 -c "print ( 'Mismatch between fpu configurations in application and spm, CONFIG_FPU=y, spm_CONFIG_FPU=n' ) ; exit ( 1 ) "
Mismatch between fpu configurations in application and spm, CONFIG_FPU=y, spm_CONFIG_FPU=n
```